### DISCUSSION TITLE
Jamulus > Zoom > Facebook?

### DISCUSSION THREAD

**Discussion by ACP-MusicMakers** [jamulussoftware/jamulus#1593] [Technical Support & Help]
Hi. New here and to Jamulus.  Not used it yet.

I host music open mics on Facebook and have a band (that uses Jamulus to practice) which want to perform, with each member in a separate location.  Therefore, my questions.

Typically, all my performers are in a Zoom mtg and I stream that meting to my Facebook group.  Does Jamulus allow for a single broadcast signal to be sent to a service like Zoom?

Thanks in advance!
AC

**Comment by hoffie** [comment]
Hi!
the [Tips and Tricks](https://jamulus.io/wiki/Tips-Tricks-More) section as a high-level description how to do that. Basically, the signal that one usually hears via headphone can be routed into other software. The actual implementation depends on the platform.
What platform do you use (Windows/Linux/Mac)?

**Reply by ACP-MusicMakers** [reply]
Thanks much.  Reviewed the Tips/Tricks, but did not see what I am looking for..  I'm using Win 10.

**Reply by dcorson-ticino-com** [reply]
I have used OBS once with Jitsi, would assume that it works with Zoom too.  I have forgotten the details in the mean time, I think I was using Jack and QJackCtl for the routing between Jamulus and OBS.
The people I play with decided, however, that just using Jamulus straight without video was the better solution and is certainly much more reliable, but the goal wasn't world-wide distribution.
Using Win10 one is somewhat restricted, it is more difficult to find routing solutions that work.

**Reply by ngocdh** [reply]
For Windows 10, the following setup works: Zoom + Voicemeeter + OBS.

OBS can publish directly to Facebook using private key, but remember there will be about 30s of delay. And you might have to adjust manually so that video from zoom and audio match.
